library/src/main/java/com/pixplicity/htmlcompat/HtmlCompat.java

```diff
@@ -246,6 +260,21 @@ public static Spanned fromHtml(@NonNull Context context, @NonNull String source,
     public static Spanned fromHtml(@NonNull Context context, @NonNull String source, int flags,
                                    @Nullable ImageGetter imageGetter, @Nullable TagHandler tagHandler,
                                    @Nullable SpanCallback spanCallback) {
+        return fromHtml(context, source, flags, imageGetter, tagHandler, spanCallback, (String[]) null);
+    }
+
+    /**
+     * Returns displayable styled text from the provided HTML string. Any &lt;img&gt; tags in the
+     * HTML will use the specified ImageGetter to request a representation of the image (use null
+     * if you don't want this) and the specified TagHandler to handle unknown tags (specify null if
+     * you don't want this). Use tagOverrides to handle any specific tags you want to handle yourself.
+     * <p>
+     * <p>This uses TagSoup to handle real HTML, including all of the brokenness found in the wild.
+     */
+    public static Spanned fromHtml(@NonNull Context context, @NonNull String source, int flags,
+            @Nullable ImageGetter imageGetter, @Nullable TagHandler tagHandler,
+            @Nullable SpanCallback spanCallback, @Nullable String... tagOverrides) {
```

Comment:
we can use varargs to be able specify more than one tag we would like to override